corehq/apps/userreports/pillow.py

```diff
@@ -301,10 +321,16 @@ def get_kafka_ucr_pillow(pillow_id='kafka-ucr-main', ucr_division=None,
     )
 
 
-def get_kafka_ucr_static_pillow(pillow_id='kafka-ucr-static', ucr_division=None,
-                                include_ucrs=None, exclude_ucrs=None, topics=None):
+def get_kafka_ucr_static_pillow(pillow_id='kafka-ucr-static', params=None):
```

Comment:
how come you went with passing in one `params` object over individual kwargs? looks like that's how it was previously